internships/Outreachy_2023-05/2023-06-09_meeting.md

```diff
@@ -0,0 +1,60 @@
+# Outreachy meeting
+
+### scikit-image project: narrative documentation
+
+- **Time:** Friday, Jun 9, 2023, 17:00–17:45 UTC
+- **[Join video call via Zoom](https://us06web.zoom.us/j/88060567580?pwd=THRpaWFnSFNwK0Fycy9FVk5RYnV5UT09)**
+- **Notes:** https://hackmd.io/@mkcor/outreachy-meeting
+- **[Archive](https://github.com/scikit-image/skimage-archive/internships/Outreachy_2023-05/)**
+
+**Present:** Ananya, Biola, Lars, Marianne
+
+**Moderator:** Marianne 
+
+
+## Agenda
+
+### Review of the week
+
+- How did it go wrt Week 2 from [Timeline](https://hackmd.io/@mkcor/outreachy-roadmap)?
+    - Interns didn't even have to be 'productive' yet!
+    - Read to learn *and* code to practise.
+    - It's motivating to have a specific goal (see next item).
+- Finish inpainting example [#6853](https://github.com/scikit-image/scikit-image/pull/6853)?
+    - pressure vs. drive
+    - [ ] Marianne adds last visualization and credits Biola.
```

Comment:
```suggestion
    - [x] Marianne adds last visualization and credits Biola.
```

Reply:
This implies that the task was already done at the time of the meeting? I would probably try to keep the submission process of meeting notes as simple and noiseless as possible. :wink: